docs/writing-tests/test-coverage.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'Test coverage'
+---
+
+Test coverage is the practice of measuring whether existing tests fully cover your code. That means surfacing areas which aren't currently being tested, such as: conditions, logic branches, functions and variables.
+
+Coverage tests examine the instrumented code against a set of industry-accepted best practices. They act as the last line of QA to improve the quality of your test suite.
+
+![Test coverage](./test-coverage-storybook.gif)
+
+## Code instrumentation with the coverage addon
+
+Storybook provides an official [test coverage addon](https://storybook.js.org/addons/@storybook/addon-coverage). Powered by [Instanbul](https://istanbul.js.org/), which allows out-of-the-box code instrumentation for the most commonly used frameworks and builders in the JavaScript ecosystem.
+
+### Set up the coverage addon
+
+To enable test coverage with Storybook, you'll need to take additional steps to set it up properly. We recommend you go through the [test runner documentation](./test-runner.md) before proceeding with the rest of the required configuration.
```

Comment:
Maybe make it more explicitly that users should use the test-runner in order to get optimal experience? The addon does help if users want to use something like playwright directly, but it's best suited for test-runner users